### ISSUE TITLE
Equity / traffic analysis does not have the traffic results in the resulting gpkg

### ISSUE THREAD

**Issue by LiekeMeijer** [Deltares/ra2ce#231]
The results from the traffic/equity analysis, stored in the geopackage, do not have the traffic results. There are traffic results saved as .csv, but these do not have a geometry attribute (optimal_route_origin_destination_link_traffic.csv) I think they should be linked to the resulting geopackage (optimal_route_origin_destination.gpkg). Or at least we want to have a resulting file where the traffic results also have geometry so we can visualize them spatially.

This issue can be reproduced by pulling the branch "traffic_analysis/equity_example_ntbk" and by running this notebook (but do set your folder path correctly to the equity_analysis folder in the examples/data folder in ra2ce - e.g. 'C:\python\ra2ce\examples\data\equity_analysis'

**Comment by LiekeMeijer** [comment]
It happens here where it is saved to a pd dataframe instead of a gpd.GeoDataFrame: 

Equity analysis.py lines 85-108


    def _get_route_traffic(
        self, traffic_data: dict[str, AccumulatedTraffic]
    ) -> pd.DataFrame:
        def convert_to_df(dict_item: tuple[str, AccumulatedTraffic]):
            node_id, traffic_values = dict_item
            u_node, v_node = self._get_key_nodes(node_id)
            return (
                u_node,
                v_node,
                traffic_values.utilitarian,
                traffic_values.egalitarian,
                traffic_values.prioritarian,
            )

        return pd.DataFrame(
            map(convert_to_df, traffic_data.items()),
            columns=[
                "u",
                "v",
                "traffic",
                "traffic_egalitarian",
                "traffic_prioritarian",
            ],
        )

**Comment by LiekeMeijer** [comment]
Two tasks:

1. Add geometry to the .csv file that holds traffic data and/or:
2. Add traffic data to the geopackage that holds geometry
3. Create code to rank the networks edges based on their traffic values -> create ranking for 'traffic' column,  'traffic_egalitarian_column' and traffic prioritarian column.
4. Look into how the ranking was used in previous work and try to reproduce

**Comment by LiekeMeijer** [comment]
Together with Carles we investigated the output of the analysis. The main conclusions:

The optimal_route_origin_destination_link_traffic.csv does indeed not contain geometry. However, the u and v values can be matched with the geometry from origins_destinations_graph_edges. We used Bramka's old code to reproduce this in a jupyter notebook.

_for var in ['traffic', 'traffic_egalitarian']:
    gdf[var] = 0
    
    for u, v in zip(df['u'],df['v']):
        if u != v:
            gdf.loc[(gdf['u']==u) & (gdf['v']==v), var] = df.loc[(df['u']==u) & (df['v']==v), var].values[0]
            if gdf.loc[(gdf['u']==v) & (gdf['v']==u), var].values == 0:
                gdf.loc[(gdf['u']==v) & (gdf['v']==u), var] = df.loc[(df['u']==u) & (df['v']==v), var].values[0]_

For now, we keep this functionality in the equity analysis Jupyter Notebook. If we want the functionality to be integrated in RA2CE, we can prioritize this at another moment.
